lib/perl/Genome/Sys/LSF/ResourceParser.pm

```diff
@@ -79,6 +81,42 @@ sub _create_option_spec {
     return ("$option=s" => sub { my ($option_name, $value, )});
 }
 
+sub null_formatter {
+    my $value = shift;
+    return $value;
+}
+
+sub number_formatter {
+    my $value = shift;
+    return $value + 0;
+}
+
+my %formatters = (
```

Comment:
If you want to encapsulate this lookup hash, it could be made a [`state`](http://perldoc.perl.org/functions/state.html) variable inside of `_formatters`.